docs/en/reference/advanced-configuration.rst

```diff
@@ -385,6 +385,13 @@ The other options are interesting in development environment:
 
 - ``AUTOGENERATE_EVAL`` will regenerate each proxy on each request,
   but without writing them to disk.
+  
+.. note::
+
+In some cases, for example when building a container, you would like 
+to generate proxies while not having a database available.
+Then you should set ``platform`` in the connection ``params``. Otherwise 
+doctrine will try to make a connection to the database to determine it.
```

Comment:
If the error message you got when attempting to generate proxies was unclear, maybe that's what needs to be improved?

Reply:
@greg0ire Yes, transporting the hint to set the platform would also help in my opinion. Maybe this could be done of all the "db-less capable" commands, as @beberlei mentioned. The error could be catched in those commands I think.

Reply:
Yes, you are right. Sorry, I was distracted by the custom field mappings we also register.

So: yes, it would probably be possible (and as helpful as a hint in the docs) to catch this error in the generate proxies command (maybe others as well) and print an according hint.